File: datasets_v2/productivity/evermemo/compose/activtiy_setting.kt
import androidx.compose.ui.tooling.preview.Preview
import androidx.compose.foundation.background
import androidx.compose.foundation.layout.*
import androidx.compose.foundation.lazy.LazyColumn
import androidx.compose.foundation.lazy.items
import androidx.compose.material3.*
import androidx.compose.runtime.*
import androidx.compose.ui.Alignment
import androidx.compose.ui.Modifier
import androidx.compose.ui.draw.paint
import androidx.compose.ui.graphics.Color
import androidx.compose.foundation.Image
import androidx.compose.ui.res.painterResource
import androidx.compose.ui.layout.ContentScale
import androidx.compose.ui.text.style.TextAlign
import androidx.compose.ui.unit.sp
import androidx.compose.ui.unit.dp

@Composable
fun activtiy_setting() {
  Column(modifier = Modifier.background(Color(0xFFE7E7E7)).fillMaxWidth().fillMaxHeight()) {
    Spacer(modifier = Modifier.background(Color(0xFFCFCFCF)).height(1.0.dp))
    Column(modifier = Modifier.fillMaxWidth().wrapContentHeight()) {
      Row(modifier = Modifier.background(Color(0xFFE7E7E7)).fillMaxWidth().height(55.0.dp), verticalAlignment = Alignment.CenterVertically) {
        Image(painter = painterResource(id = R.drawable.ic_evernote), contentDescription = null, modifier = Modifier.size(25.0.dp, 25.0.dp), contentScale = ContentScale.Fit)
        Text(text = "Connect", modifier = Modifier.wrapContentHeight().weight(1.0f).padding(start = 10.0.dp))
        Image(painter = painterResource(id = R.drawable.ic_rightarrow), contentDescription = null, modifier = Modifier.size(20.0.dp, 20.0.dp), contentScale = ContentScale.Fit)
      }
      Spacer(modifier = Modifier.background(Color(0xFFCFCFCF)).height(1.0.dp))
      Row(modifier = Modifier.background(Color(0xFFE7E7E7)).fillMaxWidth().height(55.0.dp), verticalAlignment = Alignment.CenterVertically) {
        Image(painter = painterResource(id = R.drawable.ic_start), contentDescription = null, modifier = Modifier.size(25.0.dp, 25.0.dp), contentScale = ContentScale.Fit)
        Text(text = "New memo when start-up", modifier = Modifier.wrapContentHeight().weight(1.0f).padding(start = 10.0.dp))
        Column(modifier = Modifier.wrapContentWidth().wrapContentHeight()) {
        }
      }
      Spacer(modifier = Modifier.background(Color(0xFFCFCFCF)).height(1.0.dp))
      Row(modifier = Modifier.background(Color(0xFFE7E7E7)).fillMaxWidth().height(55.0.dp), verticalAlignment = Alignment.CenterVertically) {
        Image(painter = painterResource(id = R.drawable.ic_heart), contentDescription = null, modifier = Modifier.size(25.0.dp, 25.0.dp), contentScale = ContentScale.Fit)
        Text(text = "Rate for us", modifier = Modifier.wrapContentHeight().weight(1.0f).padding(start = 10.0.dp))
        Image(painter = painterResource(id = R.drawable.ic_rightarrow), contentDescription = null, modifier = Modifier.size(20.0.dp, 20.0.dp), contentScale = ContentScale.Fit)
      }
      Spacer(modifier = Modifier.background(Color(0xFFCFCFCF)).height(1.0.dp))
    }
    Text(text = "Proudly made by Mudlab9nCoded by Daimajia,Designed by Plidezusnmade in Beijing&Shanghai", modifier = Modifier.fillMaxWidth().fillMaxHeight().padding(bottom = 15.0.dp), color = Color(0xFF7E7E7E), fontSize = 11.sp, textAlign = TextAlign.Center)
  }
}


@Preview(showBackground = true)
@Composable
fun activtiy_setting_preview() {
    MyApplicationTheme {
        activtiy_setting()
    }
}

File: experiments/productivity/evermemo/swiftui/Activtiy_setting.swift
import SwiftUI

struct Activtiy_setting: View {
  var body: some View {
    VStack(alignment: .leading, spacing: 0) {
      Rectangle().foregroundColor(Color(red: 0.8117647058823529, green: 0.8117647058823529, blue: 0.8117647058823529, opacity: 1))
      .background(Color(red: 0.8117647058823529, green: 0.8117647058823529, blue: 0.8117647058823529, opacity: 1))
      .frame(height: 1.0)
      VStack(alignment: .leading, spacing: 0) {
        HStack(alignment: .center, spacing: 0) {
          Image("ic_evernote").resizable().scaledToFit().frame(width: 25.0, height: 25.0).frame(maxHeight: .infinity, alignment: .center)
          Text("Connect").frame(maxWidth: .infinity, alignment: .leading).frame(maxWidth: .infinity).padding(.leading, 10.0).frame(maxHeight: .infinity, alignment: .center)
          Image("ic_rightarrow").resizable().scaledToFit().frame(width: 20.0, height: 20.0).frame(maxHeight: .infinity, alignment: .center)
        }
        .background(Color(red: 0.9058823529411765, green: 0.9058823529411765, blue: 0.9058823529411765, opacity: 1))
        .frame(maxWidth: .infinity)
        .frame(height: 55.0)
        Rectangle().foregroundColor(Color(red: 0.8117647058823529, green: 0.8117647058823529, blue: 0.8117647058823529, opacity: 1))
        .background(Color(red: 0.8117647058823529, green: 0.8117647058823529, blue: 0.8117647058823529, opacity: 1))
        .frame(height: 1.0)
        HStack(alignment: .center, spacing: 0) {
          Image("ic_start").resizable().scaledToFit().frame(width: 25.0, height: 25.0).frame(maxHeight: .infinity, alignment: .center)
          Text("New memo when start-up").frame(maxWidth: .infinity, alignment: .leading).frame(maxWidth: .infinity).padding(.leading, 10.0).frame(maxHeight: .infinity, alignment: .center)
          Group {
          VStack(alignment: .leading, spacing: 0) {
          }
          }
          .frame(maxHeight: .infinity, alignment: .center)
        }
        .background(Color(red: 0.9058823529411765, green: 0.9058823529411765, blue: 0.9058823529411765, opacity: 1))
        .frame(maxWidth: .infinity)
        .frame(height: 55.0)
        Rectangle().foregroundColor(Color(red: 0.8117647058823529, green: 0.8117647058823529, blue: 0.8117647058823529, opacity: 1))
        .background(Color(red: 0.8117647058823529, green: 0.8117647058823529, blue: 0.8117647058823529, opacity: 1))
        .frame(height: 1.0)
        HStack(alignment: .center, spacing: 0) {
          Image("ic_heart").resizable().scaledToFit().frame(width: 25.0, height: 25.0).frame(maxHeight: .infinity, alignment: .center)
          Text("Rate for us").frame(maxWidth: .infinity, alignment: .leading).frame(maxWidth: .infinity).padding(.leading, 10.0).frame(maxHeight: .infinity, alignment: .center)
          Image("ic_rightarrow").resizable().scaledToFit().frame(width: 20.0, height: 20.0).frame(maxHeight: .infinity, alignment: .center)
        }
        .background(Color(red: 0.9058823529411765, green: 0.9058823529411765, blue: 0.9058823529411765, opacity: 1))
        .frame(maxWidth: .infinity)
        .frame(height: 55.0)
        Rectangle().foregroundColor(Color(red: 0.8117647058823529, green: 0.8117647058823529, blue: 0.8117647058823529, opacity: 1))
        .background(Color(red: 0.8117647058823529, green: 0.8117647058823529, blue: 0.8117647058823529, opacity: 1))
        .frame(height: 1.0)
      }
      .frame(maxWidth: .infinity)
      Text("Proudly made by Mudlab9nCoded by Daimajia,Designed by Plidezusnmade in Beijing&Shanghai").font(.system(size: 11)).foregroundColor(Color(red: 0.49411764705882355, green: 0.49411764705882355, blue: 0.49411764705882355, opacity: 1)).multilineTextAlignment(.center).frame(maxWidth: .infinity, alignment: .center).padding(.bottom, 15.0)
    }
    .background(Color(red: 0.9058823529411765, green: 0.9058823529411765, blue: 0.9058823529411765, opacity: 1))
    .frame(maxWidth: .infinity, maxHeight: .infinity)
  }
}

struct Activtiy_setting_Previews: PreviewProvider {
  static var previews: some View {
    Activtiy_setting()
  }
}
